backend/controller/ingress/response.go

```diff
@@ -51,6 +51,27 @@ func ResponseBodyForVerb(sch *schema.Schema, verb *schema.Verb, body []byte, hea
 		}
 		return []byte(responseString), nil
 
+	case *schema.Int:
+		var responseInt int
+		if err := json.Unmarshal(body, &responseInt); err != nil {
+			return nil, fmt.Errorf("HTTP response body is not valid int: %w", err)
+		}
+		return []byte(fmt.Sprintf("%d", responseInt)), nil
```

Comment:
I'd use the `strconv` package rather than `Sprintf` for these values here and below.

`strconv.Itoa()`, `strconv.FormatBool()`, though `strconv.FormatFloat()` might be too annoying.

Reply:
`strconv.FormatFloat(responseFloat, 'f', -1, 64` yeah this format is crazy! :)